src/MeetingsSDKAdapter.js

```diff
@@ -273,19 +287,26 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @param {string} ID ID of the meeting to mute video
    * @memberof MeetingsSDKAdapter
    */
-  muteVideoMeeting(ID) {
+  async handleLocalVideo(ID) {
     const sdkMeeting = this.fetchMeeting(ID);
+    const state = this.meetings[ID].localVideo.getVideoTracks()[0].enabled;
```

Comment:
This could use some error handling around checking if `localVideo` is an object and has the `getVideoTracks` function.

Reply:
this object should exist when we are at this point.

Reply:
Just because it should have the proper function, doesn't mean it will. This could be a hard crash if localvideo is undefined.

Reply:
I started the discussion in the space.